toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -23,17 +24,26 @@ import (
 )
 
 const (
-	allRepoIDs             = "*"
-	builtRepoID            = "local-repo"
-	toolchainRepoId        = "toolchain-repo"
-	cacheRepoID            = "upstream-cache-repo"
-	squashChrootRunErrors  = false
-	chrootDownloadDir      = "/outputrpms"
-	leaveChrootFilesOnDisk = false
-	updateRepoID           = "mariner-official-update"
-	previewRepoID          = "mariner-preview"
-	fetcherRepoID          = "fetcher-cloned-repo"
-	cacheRepoDir           = "/upstream-cached-rpms"
+	RepoFlagMarinerDefaults = uint64(1) << iota
+	RepoFlagDownloadedCache
+	RepoFlagLocalBuilds
+	RepoFlagPreview
+	RepoFlagToolchain
+	RepoFlagUpstream
+
+	RepoFlagAll = RepoFlagToolchain | RepoFlagLocalBuilds | RepoFlagDownloadedCache | RepoFlagPreview | RepoFlagMarinerDefaults | RepoFlagUpstream
+)
+
+const (
+	chrootCloneDirContainer = "/upstream-cached-rpms"
+	chrootCloneDirRegular   = "/outputrpms"
+
+	repoIDAll            = "*"
+	repoIDBuilt          = "local-repo"
+	repoIDCacheContainer = "upstream-cache-repo"
```

Comment:
Probably not now, but these names really need to be sanitized... `upstream-cache-repo` vs `fetcher-cloned-repo`, and `/upstream-cached-rpms` vs `/outputrpms` are the same thing for different environments but have such different naming conventions.

Reply:
I agree. For now I tried to map them to more uniformly named variables. Seeing how we're moving away from using container builds, maybe we'll be able to completely remove one of these variants, since the container support was implemented only for internal use.

Reply:
Unfortunately customers are not requesting docker based builds for dev use... so they are boomaranging RIGHT back again :P